src/main/java/org/jabref/gui/preferences/XmpPrefsTab.java

```diff
@@ -68,34 +72,22 @@ public XmpPrefsTab(JabRefPreferences prefs) {
         tablePanel.setCenter(scrollPane);
 
         Button add = new Button("Add");
-        add.setPrefSize(80, 20);
-        add.setOnAction(e-> {
-            if (!addName.getText().isEmpty()) {
-                TableRow tableRow = new TableRow(addName.getText());
-                addName.clear();
-                data.add(tableRow);
-                tableRows.clear();
-                tableRows.addAll(data);
-                tableView.setItems(data);
-                tableChanged = true;
-                tableView.refresh();
+        add.setOnAction(e -> {
+            if (!StringUtil.isNullOrEmpty(bibtexFields.getSelectionModel().getSelectedItem())) {
+                XMPPrivacyFilter tableRow = new XMPPrivacyFilter(bibtexFields.getSelectionModel().getSelectedItem());
+                fields.add(tableRow);
             }
         });
         Button delete = new Button("Delete");
-        delete.setPrefSize(80, 20);
-        delete.setOnAction(e-> {
-            if (tableView.getFocusModel() != null && tableView.getFocusModel().getFocusedIndex() != -1) {
-                tableChanged = true;
+        delete.setOnAction(e -> {
+            if ((tableView.getFocusModel() != null) && (tableView.getFocusModel().getFocusedIndex() != -1)) {
```

Comment:
Why do you use the focus and not selected item ala `tableView.getSelectionModel().getSelectedItem()` ?